overlord/snapstate/snapmgr.go

```diff
@@ -780,7 +780,7 @@ func (m *SnapManager) doLinkSnap(t *state.Task, _ *tomb.Tomb) error {
 	t.Set("old-trymode", oldTryMode)
 	t.Set("old-channel", oldChannel)
 	t.Set("old-current", oldCurrent)
-	t.Set("had-candidate-in-seq", hadCandidateInSeq)
+	t.Set("added-candidate", addedCandidate)
```

Comment:
bit confused here, this is the same a snapst.Current and snapst.Candidate.Current so it's already available in undo